.travis.yml

```diff
@@ -10,6 +10,7 @@ addons:
 before_install:
   - eval $(curl https://travis-perl.github.io/init) --perl
   - echo "requires 'Code::DRY';" >> cpanfile
+  - echo "requires 'Perl::Tidy', '< 20180219';" >> cpanfile
```

Comment:
I would lock it to 20180219

Reply:
but that's the one failing

Reply:
we could use 'conflicts ..' and specify the broken version too, let's try

Reply:
LOL sorry, i meant 20180101